packages/state-manager/src/sagas/communities/updateCommunityMetadata/updateCommunityMetadata.saga.ts

```diff
@@ -22,9 +30,10 @@ export function* sendCommunityMetadataSaga(
 
   const communityMetadataPayload: CommunityMetadata = {
     id: community.id,
-    ownerCertificate: community.CA.rootCertString,
-    rootCa: community?.rootCa,
+    ownerCertificate: identity.userCertificate,
```

Comment:
This appears to be a bug that wasn't caught because we were saving the community metadata twice and the first time we saved it via the SAVE_OWNER_CERTIFICATE event where the bug didn't exist.